services/core-api/app/api/now_submissions/resources/application_list_resource.py

```diff
@@ -111,7 +111,7 @@ def _apply_filters_and_pagination(self,
     @api.doc(description='Save an application')
     @requires_role_edit_submissions
     @api.expect(APPLICATION)
-    @api.marshal_with(APPLICATION, code=201)
+    @api.marshal_with(APPLICATION, code=200)
```

Comment:
not sure why this code was changed, a 201 is a valid and expected code return for a post endpoint.

Reply:
Correct.